README.md

```diff
@@ -207,6 +207,10 @@ This will only be used if the size cannot be automatically calculated. This
 is currently only used and required if you supply a `Readable` stream as the
 file to upload. You may also use this to limit the position until which a file
 will be uploaded.
+* `agent = undefined`: specifies the [Agent](https://nodejs.org/api/http.html#http_class_http_agent)
+instance to be used to manage HTTP requests. See nodejs documentation for details.
```

Comment:
In this document we use Node.js instead of nodejs to match how they spell it on their own. Besides that, it would be nice to make it clear that this flag is only possible for the Node.js environment, and not when used inside browsers.

Reply:
Will do.

Reply:
Done.